cmd/observe/flows_test.go

```diff
@@ -174,3 +174,12 @@ func Test_getFlowsRequest_ExperimentalFieldMask_non_json_output(t *testing.T) {
 	err := handleFlowArgs(os.Stdout, filter, false)
 	assert.ErrorContains(t, err, "not compatible")
 }
+
+// Don't set the number flag in GetFlowsRequest if --input-file is specified
+func Test_getFlowsRequestWithInputFile(t *testing.T) {
```

Comment:
Can we have a test where --last is used with --input-file too?

Reply:
good idea 🥰